Jolt/Physics/Body/MotionProperties.h

```diff
@@ -52,10 +52,10 @@ class JPH_EXPORT MotionProperties
 	inline Vec3				GetAngularVelocity() const										{ JPH_ASSERT(BodyAccess::sCheckRights(BodyAccess::sVelocityAccess, BodyAccess::EAccess::Read)); return mAngularVelocity; }
 
 	/// Set world space angular velocity of the center of mass
-	void					SetAngularVelocity(Vec3Arg inAngularVelocity)					{ JPH_ASSERT(BodyAccess::sCheckRights(BodyAccess::sVelocityAccess, BodyAccess::EAccess::ReadWrite)); JPH_ASSERT(inAngularVelocity.Length() <= mMaxAngularVelocity); mAngularVelocity = inAngularVelocity; }
+	void					SetAngularVelocity(Vec3Arg inAngularVelocity)					{ JPH_ASSERT(BodyAccess::sCheckRights(BodyAccess::sVelocityAccess, BodyAccess::EAccess::ReadWrite)); JPH_ASSERT(inAngularVelocity.Length() <= mMaxAngularVelocity); mAngularVelocity = LockAngular(inAngularVelocity); }
```

Comment:
Seeing as how you're able to do this now, shouldn't `MotionProperties::ApplyForceTorqueAndDragInternal` also be calling `LockAngular`, like what's already being done for the linear velocity?

Reply:
No it shouldn't since MultiplyWorldSpaceInverseInertiaByVector already clears the velocity on the locked axis.

Reply:
Oh, my bad, I completely missed that.